src/main/java/Task.java

```diff
@@ -0,0 +1,43 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /** Total number of tasks keyed in Duke program **/
+    private static int taskCount = 0;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+        taskCount += 1;
+
+    }
+
+    public String getStatusIcon(){
+        return (isDone ? "\u2713" : "\u2718");  //return tick or X symbols
+    }
```

Comment:
Good use of camel case, very easy to identify variables and methods.

Reply:
Thank you!